lib/solidus_extension_dev_tools/templates/extension/lib/%file_name%/factories.rb.tt

```diff
@@ -0,0 +1,8 @@
+# frozen_string_literal: true
+
+FactoryBot.define do
+  # Define your Spree extensions Factories within this file to enable applications, and other extensions to use and override them.
+  #
+  # Example adding this to your spec_helper will load these Factories for use:
```

Comment:
This is already added [within the template](https://github.com/solidusio-contrib/solidus_extension_dev_tools/pull/27/files#diff-fd7c93e09ecca858fe2c49e5e6d8f945R28): remove this comment or comment-in the line in `spec_helper.rb`